Documentation/ApiOverview/CachingFramework/Configuration/Index.rst

```diff
@@ -30,7 +30,7 @@ the cache system falls back to the default backend and default frontend settings
 
 .. code-block:: php
 
-   if (!is_array($GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache'])) {
+   if (!is_array($GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['myext_mycache'] ?? false)) {
```

Comment:
Why not use `??=` here too?

Reply:
I adapted the recent version: What, if the key myext_mycache is not an array but a string or a bool? Strange, but that is the check ;-)

Reply:
Indeed, I missed the `is_array()` here. Still I don't think it's the responsibility of this code to fix a broken base IMO.